keyboards/protokeeb/keymaps/default/rules.mk

```diff
@@ -0,0 +1,2 @@
+ENCODER_MAP_ENABLE = yes
+TRI_LAYER_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
My default keymap utilizes tri-layer functionality. I believe that without this line of code in the rules.mk file, the tri-layer feature will be disabled.
https://docs.qmk.fm/features/tri_layer#tri-layers

Reply:
Nope, that rules.mk controls the behaviour of the tri-layer keycodes, not the existence of the `update_tri_layer_state` function.

Reply:
So, from what I can understand is that the function update_tri_layer_state() does not depend on TRI_LAYER_ENABLE being set in rules.mk file and It is part of QMK's core layer handling and works regardless of whether TRI_LAYER_ENABLE is enabled.

Reply:
Correct.

Reply:
Ok if that is the case I no longer need the TRI_LAYER_ENABLE build option in the rules.mk file.